src/Components/SmartComponents/SystemCves/SystemCves.js

```diff
@@ -53,16 +53,16 @@ export const CVETableContext = createContext({});
 export const SystemCVEs = ({
     entity,
     intl,
-    showHeaderLabel,
-    setPageTitle,
+    showHeaderLabel = false,
+    setPageTitle = false,
```

Comment:
This looks wrong default value. Maybe it should be a function?

Reply:
Nope, it's a boolean as listed in `propTypes`, although unfortunately named :(